docs/API.md

```diff
@@ -100,8 +100,9 @@ Basic data type: string
 
 Basic data type: object
 
-Properties:
+DS for [Delegation Signer](https://tools.ietf.org/html/rfc3658) references DNSKEY-records in the sub-delegated zone.
 
+Properties:
```

Comment:
On the next lines it says that `digest` is required but `algoritm`, `digtype` and `keytag` are optional. All of them must be required. Or?

It also says that "extra properties in *DS info* objects are ignored when present in RPC method arguments" but it should rather be that if there are extra properties, then the call is refused.

Also further down under "name sever" it says that `ip` is required, but it should be optional.

Reply:
@matsduf In the current state of the implementation of the API, `algoritm`, `digtype` and `keytag` are optional and extra properties are ignored. 
For the `ip`, i will change it in the next commit.

I think this PR is to update the API documentation that reflects its current implementation. Maybe your comments should be aggregated in a new issue to update the implementation of the API.